models/channel/src/index.ts

```diff
@@ -1,8 +1,30 @@
-export * as ChannelDraft from './channel-draft';
+import {
+  RestModelBuilder,
+  GraphqlModelBuilder,
+  CompatChannelModelBuilder,
+} from './builders';
+import * as modelPresets from './presets';
+
+// export * as ChannelDraft from './channel-draft';
```

Comment:
Can this be removed?

Reply:
Updated here: 2577856364f544d69c693a18a9fc19f38cb35672